abcclassroom/tests/test_assignment_template.py

```diff
@@ -0,0 +1,120 @@
+# Tests for assignment-template console script workflow
+
+import pytest
+import os
+from pathlib import Path
+import abcclassroom.template as abctemplate
+
+
+class TestAssignmentTemplate:
```

Comment:
i think i like the idea of individual tests rather than a class and then use fixtures. if that config item became a fixture we could use the same config for all tests across many different modules. for example the nbgrader_dir will likely need to be defined several times. that's my thought so far...